docs/source-app/api_reference/generated/lightning_app.components.python.popen.PopenPythonScript.rst

```diff
@@ -0,0 +1,14 @@
+﻿.. role:: hidden
```

Comment:
Could be removed in a follow-up PR if needed, but these files under `docs/source-app/api_reference/generated/` seem to be auto-generated. Do we need to include them, too?

Reply:
good catch, removed